src/app/mesh/qgsmeshcalculatordialog.cpp

```diff
@@ -354,10 +354,10 @@ void QgsMeshCalculatorDialog::updateInfoMessage()
   }
 }
 
-void QgsMeshCalculatorDialog::onOutputRadioButtonChange()
+void QgsMeshCalculatorDialog::onVirtualCheckboxChange()
 {
-  mOutputDatasetFileWidget->setEnabled( mOutputOnFileRadioButton->isChecked() );
-  mOutputFormatComboBox->setEnabled( mOutputOnFileRadioButton->isChecked() );
+  mOutputDatasetFileWidget->setEnabled( !mUseVirtualProviderCheckBox->isChecked() );
+  mOutputFormatComboBox->setEnabled( !mUseVirtualProviderCheckBox->isChecked() );
```

Comment:
I'm just disabling these widgets here, following the current behavior but wonder whether I should hide them instead, like in the raster calculator context.